index.js

```diff
@@ -189,6 +189,34 @@ inputTitle.value = ``;
 
 }
 
+// ************************ Code Creating Static once clicked and on a setTimeout *********************** //
+
+//selecting the image
+let mainElement = document.querySelector("img");
+// picking up the p tag
+let list = document.querySelector("p")
+// two event handlers which will handle the click on the shaking tele.
+mainElement.addEventListener("click", clickStatic)
+list.addEventListener("click", clickStatic)
+
+function clickStatic() {
+//   console.log(setTimeout(myTimeout))
+  mainElement.style.display="none";
+  list.style.display="none";
+  let errorLostScreen = document.querySelector(".tv")
+//   console.log(errorLostScreen)
+  errorLostScreen.classList.remove("no-signal");
+  errorLostScreen.classList.add("tv-static")
+  setTimeout(function() {
```

Comment:
I've really enjoyed reading this whole clickStatic() function 🥳🌟

Reply:
I'm glad, this was super fun to make 🥳